website/dbt-versions.js

```diff
@@ -18,6 +23,14 @@ exports.versions = [
 ]
 
 exports.versionedPages = [
+  {
+    "page": "docs/reference/dbt-jinja-functions/cross-database-macros",
+    "firstVersion": "1.2",
+  },
+  {
+    "page": "docs/guides/migration-guide/upgrading-to-v1.2",
+    "firstVersion": "1.2",
+  },
```

Comment:
👍 This looks right to me. I find that the versioning logic can be a bit wonky with browser settings/cookies (??) while I'm actively editing the docs site. It tends to work out in prod. _(famous last words)_

Reply:
Changing to `"page": "guides/migration/versions/upgrading-to-v1.2",` to see if that works.

Reply:
@jtcohen6 and @dbeatty10 My fix above worked, so I fixed the v1.1 page as well.

Reply:
Woohoo! Thank you, @runleonarun ! 🏆